qemu/tests/sr_iov_sanity.py

```diff
@@ -26,25 +25,26 @@ def check_network_interface_ip(interface, ipv6="no"):
     return _ip
 
 
-def ifup_down_interface(interface, action="up"):
+def ifup_down_interface(test, interface, action="up"):
     check_cmd = "ifconfig %s" % interface
     status = utils.system_output(check_cmd)
     if action == "up":
         if not check_network_interface_ip(interface):
             if "UP" in status.splitlines()[0]:
                 utils.system("ifdown %s" % interface, timeout=120,
                              ignore_status=True)
-            utils.system("ifup %s" % interface, timeout=120, ignore_status=True)
+            utils.system("ifup %s" % interface,
```

Comment:
let us remove Autotest dependency by using `process` instead of `utils` here and elsewhere